packages/client/src/pages/Game/core/entities/sprite/index.ts

```diff
@@ -19,6 +19,7 @@ export class Sprite {
   private currentAnimationFrame = 0;
   private animationFrameLimit = ANIM_FRAME_LIMIT;
   private animationFrameProgress = ANIM_FRAME_LIMIT;
+  private sizeRatio: number;
```

Comment:
```suggestion
  private readonly sizeRatio: number;
```
Можно добавить, чтобы случайно не перезаписать где-нибудь в методах